src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -48,7 +108,8 @@ def delete_container(client, container_name, fail_not_exist=False, lease_id=None
                      if_unmodified_since=None, timeout=None, bypass_immutability_policy=False,
                      processed_resource_group=None, processed_account_name=None, mgmt_client=None):
     if bypass_immutability_policy:
-        return mgmt_client.blob_containers.delete(processed_resource_group, processed_account_name, container_name)
+        mgmt_client.blob_containers.delete(processed_resource_group, processed_account_name, container_name)
+        return True
```

Comment:
why adding additional true to return here?

Reply:
As `mgmt_client.blob_containers.delete` returns None if container is deleted, `data_client.block_blob.delete_container` returns True however. So I add `return True` here to show in the same way whether add `--bypass-immutability-policy` or not.